### ISSUE TITLE
Mirror remaining Images

### ISSUE THREAD

**Issue by rmweir** [rancher/rancher#31598]
**Note:**
This will be discussed during image workshop

**Description:**
We are going to start mirroring images with the prefix "mirrored". The process to do so is as follows:

1. Create ticket at in EIO repo for new dockerhub repo​.
2. Add image to image-list at https://github.com/rancher/image-mirror.​
Example: https://github.com/rancher/image-mirror/pull/77
3. Change references of image in chart/rancher to use new image.
Example: https://github.com/rancher/charts/pull/1005

List of projects that need to be checked for mirrored images with developer that is assigned to each:

- [x] ~~Logging v1 (No images to be mirrored)~~ - @paynejacob @nickgerace (https://github.com/rancher/rancher/issues/32448)
- [x] Istio v1 - @brendarearden
- [x] Monitoring v1 - @jiaqiluo (https://github.com/rancher/rancher/issues/31871)
- [x] CIS Scan v1 - @prachidamle (https://github.com/rancher/rancher/issues/31872)
- [x] DNS - @prachidamle (https://github.com/rancher/rancher/issues/31873)
- [x] OPA Gatekeeper - @cmurphy
- [x] Backup & restore - @jiaqiluo (See comment below, not needed)
- [x] System Images including RKE and pipelines -

**Comment by dramich** [comment]
@rmweir Should also include https://github.com/rancher/rancher/blob/v2.5.7/pkg/apis/management.cattle.io/v3/tools_system_images.go

**Comment by rmweir** [comment]
@dramich this is part of RKE images since export categorizes them together and I'm working on it. I will update the issue.

**Comment by SheilaghM** [comment]
https://zube.io/rancher/rancher/c/37519
https://github.com/rancher/rancher/issues/31872

**Comment by jiaqiluo** [comment]
The rancher-backup app uses the following two images 
```
rancher/backup-restore-operator v1.0.3
rancher/kubectl v1.20.2 <-- will be in the next chart release
```
Both are rancher-owned images, so there is no need to mirror them.

**Comment by nickgerace** [comment]
Logging v1 does not have any images that need to be mirrored: https://github.com/rancher/rancher/issues/32448#issuecomment-829513975

**Comment by cmurphy** [comment]
gatekeeper is already done https://hub.docker.com/r/rancher/mirrored-openpolicyagent-gatekeeper

**Comment by cmurphy** [comment]
Unassigning myself since OPA-gatekeeper is done.

**Comment by jiaqiluo** [comment]
Unassigning myself since Monitoring v1  and Backup & restore are done.

**Comment by brendarearden** [comment]
@rmweir it looks like all the check boxes are completed, I don't know if you have more to do on this issue, but it looks like its ready to move to test.

**Comment by rmweir** [comment]
@sowmyav27 once this is in "Waiting for RC" (it isn't quite ready yet), it should be sufficient to just try to successfully download all of the rancher-images.txt.

**Comment by rmweir** [comment]
@brendarearden waiting on a couple related issue, for CIS and dns, before this can be moved.

**Comment by snasovich** [comment]
@rmweir , with https://github.com/rancher/rancher/issues/31873 in "To Test" for ~2 weeks, are we ready to have DNS box checked and move this one forward?

**Comment by deniseschannon** [comment]
It looks like all individual issues in here have been worked on. 

@rmweir Is there an easy to track if an image is mirrored and used in Rancher?

**Comment by deniseschannon** [comment]
Since each task has been validated and tracked individually, closing this tracking ticket.
